types/pwd-strength/index.d.ts

```diff
@@ -42,7 +42,7 @@ interface PwdStrengthOptions {
 interface PwdStrengthReturn {
     success: boolean;
     key: string;
-    message: string;
+    message: Array<string> | string;
```

Comment:
The CI error should be fixed by changing this type.

```suggestion
    message: string[] | string;
```

Reply:
Thank you, this change will work.

You can check the documentation here (string or string[] case):
https://github.com/rapomon/pwd-strength/#error-response
https://github.com/rapomon/pwd-strength/#error-response-with-allerrors-true-setting

Sorry but I don't have time to do a unit test for this change or follow the DefinitelyTyped rules for this change to be  published.